models/reviews.model.js

```diff
@@ -43,3 +43,28 @@ exports.selectCommentsByReviewId = (reviewId) => {
       return results.rows;
     });
 };
+
+exports.insertCommentsByReviewId = (comment, reviewId) => {
+  if (!comment.hasOwnProperty("username") || !comment.hasOwnProperty("body")) {
+    return Promise.reject({
+      status: 400,
+      message: "Missing necessary property!",
+    });
+  } else if (
+    typeof comment.username !== "string" ||
+    typeof comment.body !== "string"
+  ) {
+    return Promise.reject({
+      status: 400,
+      message: "Incorrect property type!",
```

Comment:
Could these messages be a little clearer for the user? Will a user understand what Missing property or Incorrect property means?